e2e/playwright/st_arrow_dataframe_column_types_test.py

```diff
@@ -0,0 +1,28 @@
+# Copyright (c) Streamlit Inc. (2018-2022) Snowflake Inc. (2022)
+#
+# Licensed under the Apache License, Version 2.0 (the "License");
+# you may not use this file except in compliance with the License.
+# You may obtain a copy of the License at
+#
+#     http://www.apache.org/licenses/LICENSE-2.0
+#
+# Unless required by applicable law or agreed to in writing, software
+# distributed under the License is distributed on an "AS IS" BASIS,
+# WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+# See the License for the specific language governing permissions and
+# limitations under the License.
+
+from playwright.sync_api import Page, expect
+
+from conftest import ImageCompareFunction
+
+
+def test_dataframe_column_types(
```

Comment:
Should we be duplicating these files?

Reply:
With duplicating you mean duplicating it the existing Cypress tests?

Reply:
Yep. I don't think it's a big deal though.

Reply:
Yep, I will start removing those maybe in two weeks if everything works fine with Playwright.